frontend/ui/elements/screensaver_menu.lua

```diff
@@ -38,6 +38,39 @@ return {
             genMenuItem(_("Show reading progress on sleep screen"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
             genMenuItem(_("Show book status on sleep screen"), "screensaver_type", "bookstatus", hasLastFile),
             genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
+            {
+                text = _("Ignore book cover rules"),
```

Comment:
Maybe something like

---

Show book cover:
* always
* not when finished
* not in file browser

---

Show book cover
* always
* only when reading
* only when in document